pilot/pkg/xds/xdsfake/fake.go

```diff
@@ -39,6 +39,7 @@ import (
 	kubesecrets "istio.io/istio/pilot/pkg/secrets/kube"
 	"istio.io/istio/pilot/pkg/serviceregistry"
 	kube "istio.io/istio/pilot/pkg/serviceregistry/kube/controller"
+	"istio.io/istio/pilot/pkg/xds"
```

Comment:
It must be circular import with xds, maybe try changing the xds_test.go to another package can solve this?